tests/components/elmax/test_config_flow.py

```diff
@@ -223,9 +223,25 @@ async def test_no_online_panel(hass: HomeAssistant) -> None:
 
 async def test_show_reauth(hass: HomeAssistant) -> None:
     """Test that the reauth form shows."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_ELMAX_PANEL_ID: MOCK_PANEL_ID,
+            CONF_ELMAX_USERNAME: MOCK_USERNAME,
+            CONF_ELMAX_PASSWORD: MOCK_PASSWORD,
+            CONF_ELMAX_PANEL_PIN: MOCK_PANEL_PIN,
+        },
+        unique_id=MOCK_PANEL_ID,
+    )
+    entry.add_to_hass(hass)
+
     result = await hass.config_entries.flow.async_init(
         DOMAIN,
-        context={"source": SOURCE_REAUTH},
+        context={
+            "source": SOURCE_REAUTH,
+            "unique_id": entry.unique_id,
```

Comment:
`"unique_id": entry.unique_id` appears to be missing in all other tests

Reply:
Addressed in b172033dd4. Thanks.